lib/plugins/aws/package/compile/events/websockets/lib/api.js

```diff
@@ -26,6 +38,7 @@ module.exports = {
           Description:
             this.serverless.service.provider.websocketsDescription || 'Serverless Websockets',
           ProtocolType: 'WEBSOCKET',
+          Tags: tags,
```

Comment:
We should offer it as an opt-in feature as it might cause an unexpected change for users that don't expect these tags to be populated automatically to their `websockets` APIGW. We did the similar thing in the past with applying tags to `httpApi` APIGW - please see the PR for reference - https://github.com/serverless/serverless/pull/8938/files

Reply:
@pgrzesik I understand. I will prepare similar changes as to httpApi.

Reply:
Thanks @tjurdzinski 🙇

Reply:
I added `websocketsUseProviderTags` in provider to save consistency with current setting for websockets.

```yml
provider:
  name: aws
  runtime: nodejs12.x
  websocketsApiName: custom-websockets-api-name
  websocketsApiRouteSelectionExpression: $request.body.action # custom routes are selected by the value of the action property in the body
  websocketsDescription: Custom Serverless Websockets
  websocketsUseProviderTags: true
```

Default value is false to save current flow.

Reply:
Good thinking with keeping the consistency with current approach - but I think it would be a good opportunity to actually start migrating to newer config where all websockets-related properties will be stored under `provider.websockets` namespace. I will create a separate ticket to migrate the existing ones as well

Reply:
Additionally, we should configure the deprecation in a similar way as in `httpApi` and plan to switch the default behavior in the next major version - please look into the `httpApi` PR for guidance on how that might look like or feel free to ask if something is not clear 💯

Reply:
@pgrzesik Deprecations notice added and tested locally.
Exists other way to test it? Units etc?

Reply:
There's now an error in the tests - let's ensure that all websockets tests have this setting applied - we're checking for emitted deprecations and fail tests if that's the case. 

Also, as I mentioned earlier, let's use namespaced setting under `provider.websockets` to not add more technical debt - the other properties will be migrated as a part of separate issue.